Example/Example/AnimationPreviewView.swift

```diff
@@ -6,7 +6,6 @@ import SwiftUI
 
 // MARK: - AnimationPreviewView
 
-/// TODO: Implement functionality from UIKit `AnimationPreviewViewController`
```

Comment:
Removed this since it's out of date (`AnimationPreviewViewController` no longer exists)